### ISSUE TITLE
Playback bug when playing .mov files

### ISSUE THREAD

**Issue by pumpk1nco** [iina/iina#4606]
**System and IINA version:**

- macOS  13.5.1
- IINA  1.3.3 build 138

**Expected behavior:**
By click right arrow on the keyboard should successfully forward the playback with specific time frame.

**Actual behavior:**
When forwarding the playback by clicking the right arrow the playback reset to the beginning of the video, only happens to .mov file so far and backward by left arrow works fine.

<details>
<summary><b>Crash report:</b></summary>

```
```
</details>

<br>

<details>
<summary><b>mpv log:</b></summary>

```
```
</details>

<br>

**Steps to reproduce:**

- [ ] MPV does not have this problem.

**How often does this happen?**
Every time

**Comment by low-batt** [comment]
Was the offending video taken on an iPhone?

I ask because this sounds like a duplicate of issue #4502. That issue reports that seeking forwards causes playback to stop. I wasn't quite able to reproduce that, but I did reproduce the behavior described here where seeking forwards goes backwards with a `mov` file from an iPhone 13.

The problem with seeking backwards instead of forwards is a regression. This problem has been reproduced with mpv and reported to that project in mpv issue https://github.com/mpv-player/mpv/issues/12047. IINA is waiting on a fix from the mpv project. 

If the video you are experiencing the problem with did not come from an iPhone, then that would be very interesting as so far the defect has only been seen with iPhone generated videos.

**Comment by pumpk1nco** [comment]
Yes i didn’t even realize that until you brought it up, it only happens
 with the mpv file that taken from my iPhone.
It has been bothering me for a while, when would it be fixed?

Noah Qi

**Comment by low-batt** [comment]
On this:
> when would it be fixed?

I too am wondering about that. The IINA project has to wait for the mpv project to fix this. As this works in mpv 0.34.1 I would hope that the mpv project would consider it important to correct the regression.

**Comment by Javihache** [comment]
This problem is still not fixed in IINA 1.3.4 build 140. Any estimates?

**Comment by low-batt** [comment]
I have just posted more information about this problem and a status update. See [this comment](https://github.com/iina/iina/issues/4502#issuecomment-1986987316) in issue #4502.

The quick answer is that first this problem needs to be fixed by the [FFmpeg](https://ffmpeg.org/) project. I don't have an estimate/guess as to when that project will fix the problem.

As #4606 is a duplicate of issue #4502 I'm going to close it.

**Comment by low-batt** [comment]
Duplicate of #4502
